client/my-sites/upgrades/checkout-thank-you/footer/index.jsx

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
To follow our component guidelines this should be either:

- `/checkout-thank-you-footer/index.jsx` with classes like `.checkout-thank-you-footer__`
- `checkout-thank-you/footer.jsx` with classes like `.checkout-thank-you__footer`.

(Same applies to `/header`.)